vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -154,7 +154,7 @@ NativeUIManager::NativeUIManager(winrt::Microsoft::ReactNative::ReactContext con
   if (React::implementation::QuirkSettings::GetUseWebFlexBasisBehavior(m_context.Properties()))
     YGConfigSetExperimentalFeatureEnabled(m_yogaConfig, YGExperimentalFeatureWebFlexBasis, true);
   if (React::implementation::QuirkSettings::GetMatchAndroidAndIOSStretchBehavior(m_context.Properties()))
-    YGConfigSetUseLegacyStretchBehaviour(m_yogaConfig, true);
+    YGConfigSetErrata(m_yogaConfig, YGErrataStretchFlexBasis);
```

Comment:
RN Core uses `YGErrataAll` here, so that the default configuration opts out of future breaking conformance fixes.

There is a separate API coming in Fabric to opt specific trees into `YGErrataNone`, for where we want web conformant layout.